typings/index.d.ts

```diff
@@ -1,25 +1,28 @@
 import * as React from 'react';
 
+type CB = () => void
```

Comment:
The callbacks are optional. Additionally, the code checks if they are functions and only returns them if they are. So, I think it would be more correct to go with either `Function | undefined` or `() => any | undefined`. I think `void` is too restrictive - we don't really care what the callback function returns.

The TypeScript types for React's `setState` function (which is how these callbacks are used) is typed as `() => any | undefined`